### PR TITLE
feat: adds application comment capture and submission

### PR DESCRIPTION
**Pull request by pbradley-f4**
Implements features for the following:

https://ukef-dtfs.atlassian.net/browse/DTFS2-3962
https://ukef-dtfs.atlassian.net/browse/DTFS2-3965
https://ukef-dtfs.atlassian.net/browse/DTFS2-3966
https://ukef-dtfs.atlassian.net/browse/DTFS2-3967

### CODE REVIEW

File: gef-ui/templates/application-details-comments.njk [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+      errorList: errors.errorSummary,
+      attributes: {
+        'data-cy': 'error-summary'
+      }
+    }) }}
+  {% endif %}
+
+  <div class="govuk-grid-row" data-cy="application-submit-page">
+    <div class="govuk-grid-column-two-thirds-from-desktop">
+      <p class="govuk-!-margin-bottom-6">
```
Comment (davehlewis) [inline gef-ui/templates/application-details-comments.njk:24]: There is a nunjucks component for the back button which could be used here

File: gef-ui/templates/application-details-comments.njk [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+        <p class="govuk-button-group govuk-!-margin-top-7" style="vertical-align: middle">
+          {{ govukButton({
+            text: "Submit",
+            attributes: {
+              'data-cy': 'submit-button'
+            },
+            classes: "govuk-!-margin-right-6"
+          }) }}
+          <a class="govuk-link" href="/gef/application-details/{{applicationId}}" data-cy="cancel-link">Cancel</a>
```
Comment (davehlewis) [inline gef-ui/templates/application-details-comments.njk:66]: Do any status changes need to hapeen here?

**Review by davehlewis — APPROVED** [review]
LGTM, couple of minor comments

**Review by eddieleeper — APPROVED** [review]

File: gef-ui/templates/application-details-comments.njk (hunk 2 above)
Comment (pbradley-f4) [reply gef-ui/templates/application-details-comments.njk:66]: Im not aware of any ... its simply bailing out the submission (which changes the state)

File: gef-ui/templates/application-details-comments.njk (hunk 1 above)
Comment (pbradley-f4) [reply gef-ui/templates/application-details-comments.njk:24]: the design doesnt elude this link being that component ... but a good shout to check it out.

Comment (davehlewis) [reply gef-ui/templates/application-details-comments.njk:24]: OK not a problem, hadn't looked that deeply. The position in the doc and the wording seemed to shout for that component to be used, not to worry

Comment (pbradley-f4) [reply gef-ui/templates/application-details-comments.njk]: Ok, clarified the requirement and standard backLink now in place implemented with nunjucks component. Good catch 👍

File: gef-ui/templates/application-details-comments.njk (hunk 2 above)
Comment (davehlewis) [reply gef-ui/templates/application-details-comments.njk:71]: OK that makes sense to me

File: gef-ui/templates/application-details-comments.njk (hunk 1 above)
Comment (davehlewis) [reply gef-ui/templates/application-details-comments.njk]: Nice one

**Pull request merged**